### ISSUE TITLE
New Test Case prospect: ZONEMD

### ISSUE THREAD

**Issue by tgreenx** [dotse/zonemaster#1237]
One avenue for a new Zonemaster Test Case could be focused on the ZONEMD resource record. See [RFC8976](https://datatracker.ietf.org/doc/rfc8976/). For example, it could be relevant to check its consistency across multiple name servers.

**Comment by matsduf** [comment]
Unfortunately, it will not be possible to check that the signature is valid.

**Comment by matsduf** [comment]
> I meant that it could be checked that the ZONEMD digest (or other fields) is consistent across all name servers of the apex of a zone.

Yes, and also that the values are standard compliant. I just realized that the most interesting verification is not possible (in the normal case).
